app/scenes/Collection/index.tsx

```diff
@@ -164,7 +178,22 @@ function CollectionScene() {
           ) : (
             <>
               <HeadingWithIcon>
-                <HeadingIcon collection={collection} size={40} expanded />
+                {can.update ? (
+                  <React.Suspense fallback={fallbackIcon}>
+                    <Icon
+                      icon={collection.icon ?? "collection"}
+                      color={collection.color ?? colorPalette[0]}
```

Comment:
The color fallback is encapsulated inside the `Icon` component. Maybe we should let it handle instead of providing one here?